### PR TITLE
 fix(docs-infra): update glossary to include the term template-input-variable (#39783)

### PR DESCRIPTION
**Pull request by andrewasche**
…input-variable (#41516)

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: aio/content/guide/glossary.md [hunk 1]
```diff
@@ -912,6 +912,14 @@ A TypeScript-like syntax that Angular evaluates within a [data binding](#data-bi
 
 Read about how to write template expressions in  [Template expressions](guide/template-syntax#template-expressions).
 
+{@a template-input-variable}
+
+## template input variable
+
+A template input variable is a variable you can reference within a single instance of the template. You declare a template input variable using the let keyword as in let hero.
```
Comment (atscott) [inline aio/content/guide/glossary.md:919]:
```suggestion
A template input variable is a variable you can reference within a single instance of the template. You declare a template input variable using the let keyword as in `let hero`.
```

File: aio/content/guide/glossary.md [hunk 2]
```diff
@@ -912,6 +912,14 @@ A TypeScript-like syntax that Angular evaluates within a [data binding](#data-bi
...
+
+Read and learn more about [Template Input Variables](template-reference-variables#template-input-variable).
```
Comment (atscott) [inline aio/content/guide/glossary.md:921]:
```suggestion
Read and learn more about [template input variables](template-reference-variables#template-input-variable).
```

**Review by atscott — CHANGES_REQUESTED** [review]
Hi @andrewasche, thanks for the contribution. Could you also fix the commit message? Looks like it's not quite right.

**Comment by andrewasche** [comment]
I gave it another try. Think it is getting better. I didn't get rid of the old mention to #41516, but I hope the right one sticks

**Comment by atscott** [comment]
Hi @andrewasche - your PR needed a rebase and the commit message still needed updating. I force pushed these updates to your branch, but the commit is now authored only by me. I suggest creating a new branch, making the change, and opening a new PR so you can get credit for the change. Thanks.

**Pull request closed without merging**